llvm/test/tools/llvm-exegesis/AArch64/setReg_init_check.s

```diff
@@ -0,0 +1,45 @@
+; REQUIRES: aarch64-registered-target
+
+# PPR REGISTER CLASS INITIALIZATION TESTCASE 
```

Comment:
Why are all of the comments capitalized now?

Also, generally comments are distinguished from RUN/CHECK lines by using two line start characters (eg `##` or `;;` rather than using different comment characters for different line types.